index.Rmd

```diff
@@ -103,6 +105,7 @@ This is where the results of the gdistance tests will be displayed in table form
 
 
 # References
+
```

Comment:
Overall, good job! I'm genuinely curious to see whether travel distances did differ or not and how you plan to map it.